src/api-v3/paths/payments.spec.ts

```diff
@@ -1,63 +1,52 @@
-import request from 'supertest';
-import { mockApp, rippleApi, mockedDebuglog } from "../../fixtures/mocks";
-import { expect } from 'chai';
 import sinon from 'sinon';
-import { capture } from 'ts-mockito';
-// import getAccountInfoFixture from '../../../../fixtures/getAccountInfo.json';
+import request from 'supertest';
+import { mockApp, rippleApi } from "../../fixtures/mocks";
+
 const path = '/v3/payments';
 
-describe(path, () => {
-  xit('POST - submits payment', (done) => {
-    sinon.stub(rippleApi, 'isConnected').returns(true);
-    // sinon.stub(rippleApi, 'request').resolves(getAccountInfoFixture);
+interface ErrorWithData extends Error {
+  data?: any; // eslint-disable-line @typescript-eslint/no-explicit-any
+}
 
+describe(path, () => {
+  it('returns `Account not found.` when the account does not exist', (done) => {
+    const err = new Error('Account not found.') as ErrorWithData;
+    err.name = 'RippledError';
+    err.data = { account: 'rLRnD5g6eb3TWrvfHoZ8y2mRznuu7GJzeN',
+      error: 'actNotFound',
+      error_code: 19,
+      error_message: 'Account not found.',
+      id: 3,
+      ledger_current_index: 1377457,
+      request:
+     { account: 'rLRnD5g6eb3TWrvfHoZ8y2mRznuu7GJzeN',
+       command: 'account_info',
+       id: 3 },
+      status: 'error',
+      type: 'response',
+      validated: false };
```

Comment:
The linting here is very strange.

Reply:
I think our current rules are here: https://github.com/xpring-eng/xrp-api/blob/master/.eslintrc.js

PR welcome! 😃